docs/development/getting-started.rst

```diff
@@ -41,6 +41,30 @@ You can also specify a subset of tests to run as positional arguments:
     $ # run the whole x509 testsuite, plus the fernet tests
     $ nox -e local -- tests/x509/ tests/test_fernet.py
 
+Building the docs
+-----------------
+
+Building the docs on non-Windows platforms requires manually installing
+the C library ``libenchant`` (`installation instructions`_).
+The docs can be built using ``nox``:
+
+.. code-block:: console
+
+    $ nox -e docs
+
+Troubleshooting ``libenchant`` errors on macOS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If after installing ``libenchant`` using Homebrew there is still an error saying that
```

Comment:
Is this really needed? `nox -e docs` works fine for me on an M1 mac with enchant from brew.

I'd really rather not force our docs to contain workarounds for every conceivable issue, unless its strictly necessary.

Reply:
Ah then I'm not sure. I also have an M1 with enchant from brew, and for me it doesn't work unless I use the workaround, so I assumed it would be an issue for everyone.

Out of curiosity:
- Is your `libenchant` also located at `/opt/homebrew/lib/libenchant-2.dylib` ?
- Do you have any modifications to your env variables that might be fixing the issue indirectly?

Reply:
1) yup
2) nope!

Reply:
huh weird. I removed the troubleshooting section then, only the section that mentions `libenchant` has to be manually installed remains.